### ISSUE TITLE
Putting in a pubmed ID that doesn't exist causes block

### ISSUE THREAD

**Issue by MJWainwright** [epistemonikos/isoq#177]
1. user enters a pubmed ID in add references (My Data page) that doens' exist
2. the "loading" wheel keeps spinning and there's no way to stop it
3. Even if you click on the "file upload" tab and then back to the "import from pubmed" tab the "loading" wheel is still spinning
4. the only way to stop it from spinning in order to try importing from pubmed again is to exit the project by returning to "My Workspace" and then going back into the project.

If a user enters a pubmed id that doesn't exist iSoQ needs to say "error" or "no reference found"  - not just spin endlessly.

**Comment by damian-garrido** [comment]
@MJWainwright could you check if this issue is solved on the test server please!

**Comment by MJWainwright** [comment]
@damian-garrido 

Nope, it doesn't seem fixed.  I put in the following number: 0000000000000000000 and the "find" button disappears and then nothing happens. The clear button does not appear and even if you delete the number the "find" button does not reappear. Even if you navigate to another page and come back, the "find" button is missing.

**Comment by damian-garrido** [comment]
was fixed on the last commit
